src/pages/index.tsx

```diff
@@ -6,6 +6,8 @@ import { DocumentSchemaLazy } from '@/entities/SideBar/ui/DocumentSchema.lazy';
 import { fetchSchema } from '@/shared/api/fetchSchema';
 import { Sidebar } from '@/widgets/layouts/side-bar';
 
+import homeStyles from '@/pages/home.module.css';
```

Comment:
check this attention, my be lint dont sort import.
try write npm run lint:fix in terminal